docs/UserGuide.md

```diff
@@ -53,51 +56,136 @@ JobFindr combines the rapid efficiency of Command Line Interface (CLI) with the
 [Graphical User Interface (GUI)](#user-interface), offering job application management that is suited for your specific
 needs and preferences.
 
+### First Time at JobFindr?
+
+_Welcome onboard JobFindr! Let us be part of your job search journey!_
+
+1. To get started with JobFindr, refer to the [Quick Start](#quick-start) below.
+2. After setting up JobFindr on your computer, you can get started with a [Quick Tutorial](#quick-tutorial) to get
+   familiar with the
+   features.
+3. You may also explore JobFindr at your own pace, you can refer to the [Feature](#features) section for all the
+   available features on JobFindr.
+
+### Experienced at JobFindr
+
+_Good to see you back here! Let see how we can help you?_
```

Comment:
can just change to 'Let's see how we can help you' or just 'how can we help you?'

Reply:
Yeap How can we help you sounds good